### PR TITLE
lib,cli,daemon: log the matching rule's verdict

### PR DESCRIPTION
**Pull request by jordalgo**
This is so users don't have to use the rule id to then look up the
action/verdict that was taken.

Example output:
```
[13:20:01.176233] Rule #0 matched 142 bytes with verdict CONTINUE
  TCP       : 35438 → 10057 [psh,ack]
              seq=1825888209 ack=1600908100 win=863
```
```
[13:37:50.908611] Rule #0 matched 86 bytes with verdict ACCEPT
  TCP       : 61452 → 10057 [ack]
              seq=2463015047 ack=813395332  win=406
```

Issue: https://github.com/facebook/bpfilter/issues/358

### CODE REVIEW

**Comment by jordalgo** [comment]
Need advice on adding tests for this.

File: src/libbpfilter/include/bpfilter/runtime.h [hunk 1]
```diff
@@ -97,6 +99,9 @@ struct bf_log
     /** ID of the rule triggering the log. */
     __u32 rule_id;
 
+    /** Verdict of the rule triggering the log. */
+    enum bf_verdict verdict;
```
Comment (qdeslandes) [inline src/libbpfilter/include/bpfilter/runtime.h]: Use `__32` for the verdict, we don't want to pull-in extra headers in here as this will eventually cause trouble with the ElfStubs compilation.

File: src/bpfilter/cgen/elfstub.h [hunk 2]
```diff
@@ -122,14 +122,14 @@ enum bf_elfstub_id
...
      * **Parameters**
      * - `ctx`: address of the `bf_runtime` context of the program.
-     * - `map`: pointer to the logs ring buffer.
-     * - `layers`: user-requested headers to log.
-     * - `l3_proto`: layer 3 (internet) protocol identifier.
-     * - `l4_proto`: layer 4 (transport) protocol identifier.
+     * - `rule_id`: id of the matched rule.
+     * - `headers`: user-requested headers to log.
+     * - `verdict`: verdict of the matched rule.
+     * - `l3_l4_proto`: layer 3 (internet) and layer 4 (transport) protocols packed.
```
Comment (qdeslandes) [inline src/bpfilter/cgen/elfstub.h]: Describe the bytes layout `(l3 << 16 | l4)`.

**Review by qdeslandes — CHANGES_REQUESTED** [review]
Please fix your commit titles (see https://bpfilter.io/developers/style.html), and add a brief description to the commit modifying the `bf_log` stub arguments (for future reference).

> Need advice on adding tests for this.

The `log` action is not automatically testing for now. Feel free to add an e2e test script to create a chain and parse the output of bfcli. Realistically, if this was my PR, I would manually test a few cases (including edge cases to validate byte order), and paste the results in the PR, that would be enough.

File: src/libbpfilter/include/bpfilter/runtime.h (hunk 1 above)
Comment (jordalgo) [reply src/libbpfilter/include/bpfilter/runtime.h]: Done.

File: src/bpfilter/cgen/elfstub.h (hunk 2 above)
Comment (jordalgo) [reply src/bpfilter/cgen/elfstub.h]: Done.
